internal/csv/csv.go

```diff
@@ -0,0 +1,97 @@
+package csv
+
+import (
+	"encoding/json"
+	"strings"
+
+	operatorsv1alpha1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
+	"github.com/operator-framework/operator-manifest-tools/pkg/imagename"
+	appsv1 "k8s.io/api/apps/v1"
+)
+
+const InfrastructureFeaturesAnnotation = "operators.openshift.io/infrastructure-features"
+
+// SupportsDisconnected accepts a stringified list of supported features
+// and returns true if "disconnected" is listed as a supported feature.
+//
+// E.g. '["disconnected"]'.
```

Comment:
Big super small nit: This should probably be backticks, and not single quotes, to signify a JSON literal.

Reply:
I don't think this is the case in the YAML manifests? If we wanted to represent the value in pure-go, it'd probably be 

```
`'["disconnected"]'`
```

But in the yaml, it's just single quotes. Example: https://github.com/redhat-openshift-ecosystem/community-operators-prod/blob/544d90d878a0c1dc7b194f004e00f57802578279/operators/pcc-operator/0.2.0/manifests/pcc-operator.v0.2.0.clusterserviceversion.yaml#L63

Reply:
I see. Interesting. Like I said, super small nit. But I didn't realize that's how the field was actually represented in the yaml. It seems to be so that it is escaped, and not parsed as an array in the unmarshalling. Carry on.